lib/auth/touchid/api.go

```diff
@@ -622,6 +622,37 @@ func ListCredentials() ([]CredentialInfo, error) {
 	return infos, nil
 }
 
+// CredentialExists checks if credentials exists in Secure Enclave credentials.
+// Does not require user interaction.
+func CredentialExists(credentialID, rpID string) (bool, error) {
```

Comment:
We should be able to find and delete the credential with just the credentialID, no rpID required.

Reply:
I have deleted that fn, because if I use non-interactive delete, I don't need to check for existence. Now code is more readable.